test/files/pos/CharSequence-implicits.scala

```diff
@@ -0,0 +1,4 @@
+object Test extends App {
+  Array('a', 'b'): CharSequence
+  IndexedSeq('a', 'b'): CharSequence
+}
```

Comment:
I converted it to a JUnit test, and for reasons I don't understand, even though the test is in `package scala`, it doesn't compile if the implicits in question are declared `private[scala]`. So that's good, actually, the test tests everything. (But I don't know why.)